project-base/storefront/components/Blocks/Skeleton/SkeletonManager.tsx

```diff
@@ -41,7 +41,7 @@ export const SkeletonManager: FC<SkeletonManagerProps> = ({ isFetchingData, isPa
             return <SkeletonPageProductDetailMainVariant />;
         case 'category':
         case 'seo_category':
-            return <SkeletonPageProductsList />;
+            return <SkeletonModuleProductsList />;
         case 'brand':
         case 'flag':
             return <SkeletonPageProductsListSimple />;
```

Comment:
Now it is confusing, because below we have SkeletonPage, and here we have SkeletonModule. I would use unified naming.

Reply:
Yup, already solved by refactoring.